crocket/src/main/java/com/crocket/model/DirectionLine.java

```diff
@@ -2,17 +2,15 @@
 
 public class DirectionLine {
     private int degreeAngle;
-    private double height;
-    private double width;
+    private final double height = 100;
```

Comment:
I would suggest changing these to `HEIGHT` and `WIDTH` since they are constants, to follow naming conventions